lightning/src/ln/channel.rs

```diff
@@ -4303,7 +4317,22 @@ impl<Signer: Sign> Channel<Signer> {
 
 	/// Allowed in any state (including after shutdown)
 	pub fn get_counterparty_htlc_minimum_msat(&self) -> u64 {
```

Comment:
Decided to keep the return type of this function as `u64`, and not bleed out to the rest of the project that this is now stored as a `Option<u64>` type, and return `0` in case it doesn't exist as this was the value it was initially set to previously in the `new_outbound` function. The only time the `outbound_htlc_minimum_msat` shouldn't exist now is when when an outbound channel has been created which hasn't been accepted by the peer yet.

I will of course change this if you think it's more appropriate that this function returns a `Option<u64>` type.